smoothie.d.ts

```diff
@@ -4,185 +4,189 @@
+     *
+     * @param timestamp the position, in time, of this data point
+     * @param value the value of this data point
+     * @param sumRepeatedTimeStampValues if <code>timestamp</code> has an exact match in the series, this flag controls
+     * whether it is replaced, or the values summed (defaults to false.)
+     */
+    append(timestamp: number, value: number, sumRepeatedTimeStampValues?: boolean): void;
+
+    dropOldData(oldestValidTime: number, maxDataSetLength: number): void;
+}
+
+export interface IGridOptions {
+    /** The background colour of the chart. */
+    fillStyle?: string;
+    /** The pixel width of grid lines. */
+    lineWidth?: number;
+    /** Colour of grid lines. */
+    strokeStyle?: string;
+    /** Distance between vertical grid lines. */
+    millisPerLine?: number;
+    /** Controls whether grid lines are 1px sharp, or softened. */
+    sharpLines?: boolean;
+    /** Number of vertical sections marked out by horizontal grid lines. */
+    verticalSections?: number;
+    /** Whether the grid lines trace the border of the chart or not. */
+    borderVisible?: boolean;
+}
+
+export interface ILabelOptions {
+    /** Enables/disables labels showing the min/max values. */
+    disabled?: boolean;
+    /** Colour for text of labels. */
+    fillStyle?: string;
+    fontSize?: number;
+    fontFamily?: string;
+    precision?: number;
+}
+
+export interface IRange { min: number; max: number }
+
+export interface IHorizontalLine {
+    value?: number;
+    color?: string;
+    lineWidth?: number;
+}
+
+export interface IChartOptions {
+    /** Specify to clamp the lower y-axis to a given value. */
+    minValue?: number;
+    /** Specify to clamp the upper y-axis to a given value. */
+    maxValue?: number;
+    /** Allows proportional padding to be added above the chart. for 10% padding, specify 1.1. */
+    maxValueScale?: number;
+    yRangeFunction?: (range:IRange)=>IRange;
+    /** Controls the rate at which y-value zoom animation occurs. */
+    scaleSmoothing?: number;
+    /** Sets the speed at which the chart pans by. */
+    millisPerPixel?: number;
+    /** Whether to render at different DPI depending upon the device. Enabled by default. */
+    enableDpiScaling?: boolean;
+    yMinFormatter?: (min:number, precision:number)=>string;
+    yMaxFormatter?: (max:number, precision:number)=>string;
+    maxDataSetLength?: number;
+    /** One of: 'bezier', 'linear', 'step' */
+    interpolation?: string;
+    /** Optional function to format time stamps for bottom of chart. You may use <code>SmoothieChart.timeFormatter</code>, or your own/ */
+    timestampFormatter?: (date:Date)=>string;
+    horizontalLines?: IHorizontalLine[];
+
+    grid?: IGridOptions;
+
+    labels?: ILabelOptions;
+}
+
+/**
+ * Initialises a new <code>SmoothieChart</code>.
+ *
+ * Options are optional and may be sparsely populated. Just specify the values you
+ * need and the rest will be given sensible defaults.
+ */
```

Comment:
Just noticed this comment looks like it's supposed to be on the constructor (the problem existed before this PR).